content/reference-ledger-format.md

```diff
@@ -303,6 +305,72 @@ The `index` of an Offer node is the SHA-512Half of the following values put toge
+| `Amount`            | String    | Amount            | Total XRP, in drops, that has been allocated to this channel. This includes XRP that has been paid to the destination address. This is initially set by the transaction that created the channel and can be increased if the source address sends a PaymentChannelFund transaction. |
+| `Balance`           | String    | Amount            | Total XRP, in drops, currently stored in the channel. This XRP can still be paid to the destination address with PaymentChannelClaim transactions. If the channel closes, this XRP is returned to the source address. |
+| `PublicKey`         | String    | PubKey            | Public key, in hexadecimal, of the key pair that can be used to sign claims against this channel. This can be any valid secp256k1 or Ed25519 public key. This is set by the transaction that created the channel and must match the public key used in claims against the channel. The channel source address can also send XRP to the destination without signed claims. |
+| `SettleDelay`       | Number    | UInt32            | Number of seconds the source address must wait to close the channel if it still has any XRP in it. Smaller values mean that the destination address has less time to redeem any outstanding claims after the source address requests to close the channel. Can be any value that fits in a 32-bit unsigned integer (0 to 2^32-1). This is set by the transaction that creates the channel. |
+| `OwnerNode`         | String    | UInt64            | A hint indicating which page of the source address's owner directory links to this node, in case the directory consists of multiple nodes. |
+| `PreviousTxnID`     | String    | Hash256           | The identifying hash of the transaction that most recently modified this node. |
+| `PreviousTxnLgrSeq` | Number    | UInt32            | The [index of the ledger](#ledger-index) that contains the transaction that most recently modified this node. |
+| `Flags`             | Number    | UInt32            | A bit-map of boolean flags enabled for this payment channel. Currently, the protocol defines no flags for `PayChannel` nodes. |
+| `Expiration`        | Number    | UInt32            | _(Optional)_ The mutable expiration time for this payment channel, in [seconds since the Ripple Epoch](reference-rippled.html#specifying-time). The channel is expired if this value is present and smaller than the previous ledger's [`close_time` field](#header-format). This starts out omitted, and can be added, removed, or updated by the source address with a PaymentChannelFund or PaymentChannelClaim transaction. When added or updated, this must always be at least `SettleDelay` seconds after the previous ledger's [`close_time` field](#header-format). |
```

Comment:
Setting the `Expiration` from the `ChannelFund` tx works as follows:
The requested expiration must be greater than the minimum of the current `Expiration` (if any) and the `prev_close + settle_delay`. Notice this means you can set an expiration less than `prev_clost + settle_delay`, but only if `Expiration` has already been set. If the tx tries to set an invalid expiration, the tx fails with `temBAD_EXPIRATION`.

The rules for setting `Expriation` in `Claim` are as follows:
1) Calculate a proposed expiration as `prev_close + settle_delay`.
2) If the proposed expiration is less than the current expiration (if any), use the proposed expiration, otherwise keep the current expiration.